src/features/comments.ts

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { createComment, deleteComment, getPostComments } from '../api/comments';
+import { Comment } from '../types/Comment';
+
+const initialState = {
+  comments: [] as Comment[],
+  hasError: false,
+  loaded: true,
+};
+
+export const setComments = createAsyncThunk(
+  'comments/setComments',
+  (userId: number) => {
+    return getPostComments(userId);
+  },
+);
+
+export const addComment = createAsyncThunk(
+  'comments/add',
+  // (data: Omit<Comment, 'id'>) => {
+  //   return createComment(data);
+  // },
+  async (comment: Omit<Comment, 'id'>) => {
+    const value = await createComment(comment);
+
+    return value;
+  },
+);
+export const removeComment = createAsyncThunk(
+  'comments/delete',
+  // (commentId: number) => {
+  //   return deleteComment(commentId);
+  // },
```

Comment:
These lines are commented out. If they are not needed, consider removing them to keep the code clean. If they are intended for future use, ensure there's a clear comment explaining their purpose.